scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/AerieController.java

```diff
@@ -482,16 +483,7 @@ public boolean handleResponse(JSONObject response) {
 
   private ActivityInstance jsonToInstance(JSONObject jsonActivity) {
     String type = jsonActivity.getString("type");
-    var actTypes = missionModelWrapper.getMissionModel().getTaskSpecificationTypes();
-    var specType = actTypes.get(type);
-    if (specType == null) {
-      throw new IllegalArgumentException("Activity type is not present in mission model");
-    }
-    var schedulerActType = missionModelWrapper.getActivityType(type);
-    if (schedulerActType == null) {
-      throw new IllegalArgumentException("Activity type is not present in scheduler mission model wrapper");
-    }
-
+    var schedulerActType = activityTypes.get(type);
```

Comment:
Should there still be a check on if the activity type is valid, throwing the `IllegalArgumentException` when it isn't? As written, if the type isn't known, this will just cause a runtime exception.

Reply:
You're right, I've added your suggestion.